app/_src/gateway/plugin-development/pdk/kong.plugin.md

```diff
@@ -30,8 +30,14 @@ Returns the instance ID of the plugin.
 
 **Usage**
 
+{% if_version lte:3.5.x %}
 ``` lua
 kong.request.get_id() -- "123e4567-e89b-12d3-a456-426614174000"
 ```
+{% endif_version %}
 
-
+{% if_version gte:3.6.x %}
+``` lua
+kong.plugin.get_id() -- "123e4567-e89b-12d3-a456-426614174000"
+```
+{% endif_version %}
```

Comment:
This one is a doc fix, not a change to the PDK: https://github.com/Kong/kong/pull/12131

So we can just fix it and don't need to version tag it.